
Make the exposed API for Darwin similar to that on the other platforms, so
applications using it will be portable.

  - Hans Petter Jansson <hpj@novell.com>

diff -upr howl-0.9.8-orig/src/lib/howl/MacOSX/macosx_salt.c howl-0.9.8/src/lib/howl/MacOSX/macosx_salt.c
--- howl-0.9.8-orig/src/lib/howl/MacOSX/macosx_salt.c	Fri Jan 28 11:27:36 2005
+++ howl-0.9.8/src/lib/howl/MacOSX/macosx_salt.c	Fri Jan 28 11:32:09 2005
@@ -229,3 +229,19 @@ sw_socket_error_code()
 {
 	return errno;
 }
+
+
+sw_result
+sw_salt_lock(
+				sw_salt	self)
+{
+	sw_assert(self);
+}
+
+
+sw_result
+sw_salt_unlock(
+				sw_salt	self)
+{
+	sw_assert(self);
+}
